__mocks__/@pagopa/io-react-native-jwt/index.ts

```diff
@@ -0,0 +1,22 @@
+// mock react native bridged modules
+
+import { sha256 } from "js-sha256";
+export * from "@pagopa/io-react-native-jwt";
+export const sha256ToBase64 = (v: string) =>
+  removePadding(hexToBase64(sha256(v)));
+
+function hexToBase64(hexstring: string) {
+  const x = hexstring.match(/\w{2}/g) || [];
+  const g = x
+    .map(function (a) {
+      return String.fromCharCode(parseInt(a, 16));
+    })
+    .join("");
+
+  return btoa(g);
+}
+
+function removePadding(encoded: string): string {
```

Comment:
What do you think about importing it from `io-react-native-jwt`?

Reply:
I don't think it's an exported method

Reply:
Ok, no problem